srml/babe/src/lib.rs

```diff
@@ -42,9 +39,15 @@ use inherents::{RuntimeString, InherentIdentifier, InherentData, ProvideInherent
 #[cfg(feature = "std")]
 use inherents::{InherentDataProviders, ProvideInherentData};
 use babe_primitives::{BABE_ENGINE_ID, ConsensusLog, BabeAuthorityWeight, Epoch, RawBabePreDigest};
-pub use babe_primitives::{AuthorityId, VRF_OUTPUT_LENGTH, PUBLIC_KEY_LENGTH};
+pub use babe_primitives::{sr25519::AuthorityId, VRF_OUTPUT_LENGTH, PUBLIC_KEY_LENGTH};
 use system::ensure_root;
 
+#[cfg(all(feature = "std", test))]
+mod tests;
+
+#[cfg(all(test, feature = "std"))]
```

Comment:
```suggestion
#[cfg(all(feature = "std", test))]
```

Nitpick, but the different ordering here and above was triggering my OCD.